src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute of the link tag should point to a CSS file, not an SCSS file. Browsers can't interpret SCSS files. You should compile your SCSS to CSS and then link the CSS file here.